lib/types/user-types.ts

```diff
@@ -3,3 +3,8 @@ export interface FormOwner {
   email: string;
   active: boolean;
 }
+
+export enum UserRole {
```

Comment:
This would be redundant if the enums feature is used in prisma. https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-enums.